proposals/utils/stepper.py

```diff
@@ -262,7 +262,7 @@ def get_form_errors(self, exclude_submit=False):
                 # When using this method on the submit page, we
                 # use the form's own validation, otherwise it gets
                 # validated twice
-                if exclude_submit and item.form_class == ProposalSubmitForm:
+                if exclude_submit and getattr(item, "form_class", None) == ProposalSubmitForm:
```

Comment:
Test for equal identity of class types should be `x is some_class` not `x == some_class`. The latter is for eqaulity of value and is not well-defined for Forms.

We missed both these issues on the first pass, rip